src/mapml/layers/MapMLLayer.js

```diff
@@ -562,12 +566,12 @@ export var MapMLLayer = L.Layer.extend({
                   zIndex++;
                 }
                 controls.classList.remove("mapml-draggable");
-                document.body.onmousemove = document.body.onmouseup = null;
+                document.body.ontouchmove = document.body.onmousemove = document.body.ontouchend = document.body.onmouseup = null;
```

Comment:
I am wondering if we should be reviewing everywhere that we are using mouse events, should we review if touch events are necessary??

Reply:
Yes I think it is definitely necessary but not urgent, I did not find other bugs/issues except the layer control ones (which I have fixed on this branch) during the debugging process! :)